test/services/fixtures/create-user.dto.ts

```diff
@@ -46,7 +46,8 @@ export class CreateUserDto {
 
   @ApiProperty({
     type: String,
-    isArray: true
+    isArray: true,
+    format: 'uri'
```

Comment:
This fixture did not previously define a `format`, but it looked like an appropriate place to demonstrate the expected behavior.